cmd/celestia-appd/cmd/init.go

```diff
@@ -0,0 +1,21 @@
+package cmd
+
+import (
+	"github.com/cosmos/cosmos-sdk/x/genutil/client/cli"
+	"github.com/spf13/cobra"
+
+	"github.com/celestiaorg/celestia-app/v3/app"
+)
+
+// InitCmd returns a custom version of the default init command with improved help docs.
+func InitCmd() *cobra.Command {
+	cmd := cli.InitCmd(app.ModuleBasics, app.DefaultNodeHome)
+
+	cmd.Short = "Initialize configuration files for a Celestia consensus node"
+	cmd.Long = `
+This command sets up a genesis file and default configuration for either a validator or a full node.`
```

Comment:
nit: this doesn't need to differentiate between a validator or consensus node. Also doesn't need a new line.

```suggestion
	cmd.Long = "This command sets up a genesis file and default configuration for either a validator or a full node."
```